lib/postgrator-cli.js

```diff
@@ -23,16 +23,20 @@ function getMigrateToNumber(toArgument) {
 }
 
 function getAbsolutePath(fileOrDirectory) {
-    const absPath = (path.isAbsolute(fileOrDirectory))
-        ? fileOrDirectory
-        : path.join(process.cwd(), fileOrDirectory);
-
-    // Paths must use posix forward slashes to work properly ever since
-    // postgrator updated to the newer version of glob
-    //
-    // https://github.com/rickbergfalk/postgrator/blob/master/CHANGELOG.md#800
-    // https://github.com/isaacs/node-glob/blob/main/changelog.md#81
-    return absPath.replaceAll(path.sep, path.posix.sep);
+    if (!Array.isArray(fileOrDirectory)) fileOrDirectory = [fileOrDirectory];
```

Comment:
We could use the `arrify` package to make this look a bit neater

```
arrify(fileOrDirectory).map(...)
```

Reply:
It's just another external dependency. I'm of the opinion that the fewer external dependencies the better. Or you need to be sure of the external package you are using. If you are sure about this package, then I will add it.